src/doom/d_main.c

```diff
@@ -2151,6 +2151,16 @@ void D_DoomMain (void)
 
     PrintGameVersion();
 
+    p = M_CheckParmWithArgs("-playmusic", 1);
+    if (p)
+    {
+        DEH_printf("Now playing %s\n", myargv[p+1]);
+        S_PlayMusicLump(myargv[p+1]);
+        while (I_MusicIsPlaying());
```

Comment:
Please do something in this loop, else you will end up with 100% CPU consumption. E.g.
```
while (I_MusicIsPlaying()) {sleep(1)};
```